src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,12 @@
 
 public class MainApp {
 
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#please-dont-add-redundant-empty-lines-to-your-code)